README.md

```diff
@@ -0,0 +1,52 @@
+# CertificatesRole Name
+
+This role install a Certificate Authoriy and generates certificates for every host in the inventory and copys them over
+
+## Requirements
+
+This role requires and apt based system
+
+
+## Role Variables
+
+### General
+
+| Name                              | Required/Default              | Description                                                    |
+|-----------------------------------|:-----------------------------:|----------------------------------------------------------------|
+| certificates_ca_server            | :heavy_check_mark:            | Inventory Hostname of the server running the ca                |
+| certificates_ca_key_dir           | `/etc/ssl/private/ansible-ca` | Directory where the files for the ca should be stored          |
+| certificates_local_ca_path        | `/etc/ssl/ansible-ca.crt`     | Path where the ca.crt should be copied to on the localhost     |
+| certificates_ca_organization_name | :heavy_check_mark:            | Organization Name running the CA                               |
+| certificates_ca_common_name       | :heavy_check_mark:            | Common Name for the CA                                         |
+| certificates_ca_country_name      | :heavy_check_mark:            | Country where the CA ist located                               |
+| certificates_ca_email_address     | :heavy_check_mark:            | Email Address where the CA Administrator can be reached        |
+| certificates_local_key_dir        | `/etc/ssl/private/ansible`    | Directory where the files for the localhost should be stored   |
+| certificates_common_name          | `{{ inventory_hostname }}`    | Common Name for the certificate for the localhost              |
+| certificates_country_name         | :heavy_check_mark:            | Country where the Host is located                              |
+| certificates_email_address        | :heavy_check_mark:            | Email Address where the localhost Administrator can be reached |
+| certificates_organization_name    | :heavy_check_mark:            | Organization Name that runs the localhost                      |
```

Comment:
Is there any way I can set key the length?

Reply:
now there is